### ISSUE TITLE
Expose ignoreIncludes as an option for mjml2html

### ISSUE THREAD

**Issue by emilyfeder** [mjmlio/mjml#1886]
**Is your feature request related to a problem? Please describe.**
ignoreIncludes is an option of MJMLParser, but you can't pass in that option when using mjml's default mjml2html export. For my node.js project, I would like to turn off includes for all my templates easily instead of going through all them and manually removing mj-include statements.

**Describe the solution you'd like**
Accept ignoreIncludes as an option to mjml2html and pass it along to be used when calling MJMLParser.
```
import mjml2html from 'mjml'
const output = mjml2html(`
<mjml>
  <mj-body>
    <mj-include path="./header" /> <!-- or 'header.mjml' -->
  </mj-body>
</mjml>
`, {
    ignoreIncludes: true
});

```

**Describe alternatives you've considered**
I've considered importing MJMLParser myself and passing the output to mjml2html myself, but I would be worried that the interface could change on me without being aware since MJMLParser is not part of the official documentation.

**Additional context**
I'm using mjml v4.6.2

**Comment by iRyusa** [comment]
This was an option in MJML3, dunno why we no longer expose it to be honest cc @kmcb777 can we add this for next release ?

**Comment by iRyusa** [comment]
It's now available in 4.6.3 👍

**Comment by emilyfeder** [comment]
Thanks for the quick fix!
